src/App.tsx

```diff
@@ -15,141 +88,78 @@ export const App: React.FC = () => {
-            <form>
-              <input
-                type="text"
-                className="todo__title-field"
-                placeholder="Empty todo will be deleted"
-                value="Todo is being edited now"
-              />
-            </form>
-
-            <div className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-
-          {/* This todo is in loadind state */}
-          <div className="todo">
-            <label className="todo__status-label">
-              <input type="checkbox" className="todo__status" />
-            </label>
-
-            <span className="todo__title">Todo is being saved now</span>
-            <button type="button" className="todo__remove">×</button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        {todos.length > 0 && (
+          <>
+            <section className="todoapp__main">
+              <TodoList todos={filteredTodos} />
+            </section>
+
+            <footer className="todoapp__footer">
+              <span className="todo-count">
+                {notCompletedTodos.length === 1 ? '1 item left' : `${notCompletedTodos.length} items left`}
+              </span>
+
+              <TodoFilter filter={filter} setFilter={setFilter} />
+
+              {completedTodos.length > 0 ? (
+                <button
+                  type="button"
+                  className="todoapp__clear-completed"
+                >
+                  Clear completed
+                </button>
+              ) : (
+                <button
+                  style={{ visibility: 'hidden' }}
+                  type="button"
+                  className="todoapp__clear-completed"
+                >
+                  Clear completed
+                </button>
+              )}
```

Comment:
No need to render button and hide it
```suggestion
              {completedTodos.length > 0 ? (
                <button
                  type="button"
                  className="todoapp__clear-completed"
                >
                  Clear completed
                </button>
              ) : null}
```